scala/AdditionalRemarks.md

```diff
@@ -0,0 +1,51 @@
+Originate's Scala Style Guide
```

Comment:
can we call it "Originate's Scala Guide" since it's more than just style now (here and at the top of all sections)